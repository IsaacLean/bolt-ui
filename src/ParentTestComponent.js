import React from 'react';

import TestComponent from './TestComponent';

class ParentTestComponent extends React.Component {
  render () {
    const style = {color: this.props.color};
    return <div>
      <h1 style={ style }>
        HELLO WORLD!
      </h1>
      <TestComponent text="Lorem Ipsum" color="blue" />
    </div>;
  }
}

export default ParentTestComponent;
