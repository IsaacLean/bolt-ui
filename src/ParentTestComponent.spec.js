import React from 'react';
import ParentTestComponent from './ParentTestComponent';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

test('Empty ParentTestComponent snapshot', () => {
  const component = shallow(<ParentTestComponent />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});
