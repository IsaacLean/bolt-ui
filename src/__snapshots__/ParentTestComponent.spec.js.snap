// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty ParentTestComponent snapshot 1`] = `
<div>
  <h1
    style={
      Object {
        "color": undefined,
      }
    }
  >
    HELLO WORLD!
  </h1>
  <TestComponent
    color="blue"
    text="Lorem Ipsum"
  />
</div>
`;
